src/app/containers/ConsentBanner/Canonical/logic.js

```diff
@@ -0,0 +1,81 @@
+import 'isomorphic-fetch';
+import Cookie from 'js-cookie';
+
+const PRIVACY_COOKIE = 'ckns_privacy';
+const EXPLICIT_COOKIE = 'ckns_explicit';
+const POLICY_COOKIE = 'ckns_policy';
+const COOKIE_EXPIRY = 365;
+const BANNER_APPROVED = '1';
+const POLICY_APPROVED = '111';
+const POLICY_DENIED = '000';
+
+const onClient = typeof document !== 'undefined';
+
+const setCookie = (name, value) =>
+  Cookie.set(name, value, { expires: COOKIE_EXPIRY });
+
+const cookieOven = (cookieOvenUrl, value) => {
+  try {
+    fetch(`${cookieOvenUrl}/${value}`);
+  } catch (e) {} // eslint-disable-line no-empty
```

Comment:
This catch is empty, rather than logging, to allow the file to be reusable. Not 100% on it, but its a very inconsequential fetch